tools/test.py

```diff
@@ -333,7 +333,7 @@ def HasRun(self, output):
       (total_seconds, duration.microseconds / 1000))
     if self.severity is not 'ok' or self.traceback is not '':
       if output.HasTimedOut():
-        self.traceback = 'timeout'
+        self.traceback = 'timeout\n' + output.output.stdout + output.output.stderr
```

Comment:
I suggest to add a line break between stdout and stderr.

Reply:
The way it's done here is the way it is done on line 291. Wanted to keep it consistent. Maybe a subsequent PR can change them both if there's agreement that a blank line would help.